tools/android_lint/bin/main.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart' as path;
+import 'package:process/process.dart';
+
+/// Runs the Android SDK Lint tool on flutter/shell/platform/android.
```

Comment:
When will this run?  And what will be the impact on the fact that most files will not conform to our desired styles?

Reply:
Right now it would be run by hand.  Longer term we want to run it as part of CI.

Reply:
If we determine that the linter is suggesting things that violate our desired styles we can turn them off.  For now we want to just try to catch as much as we can.

Reply:
But these aren't being run on PRs are they?  What I want to avoid is being forced to make changes to existing files just because I had to alter a constructor argument, etc.